bitnami/discourse/values.yaml

```diff
@@ -68,6 +68,10 @@ discourse:
   ##
   # password:
 
+  ## Wheter to use an existing secret to store the Discourse passwords
```

Comment:
I think we can make it clearer by stating something like "Name of an existing secret containing the password (ignores previous `password`)"

Reply:
done